fury/ui.py

```diff
@@ -1100,11 +1100,18 @@ def __init__(self, text="Text Block", font_size=18, font_family='Arial',
             Makes text italicised.
         shadow : bool
             Adds text shadow.
+        size : (int, int)
+            Size (width, height) in pixels of the Text.
```

Comment:
Hello @skoudoro , travis CI is failing because of the text overflow fix that I did in #173 . There I used the `self.size` parameter and now as I have changed the defination of `_get_size`, its failing. I will get that fixed no problem.